pydantic/validators.py

```diff
@@ -386,10 +394,7 @@ def callable_validator(v: Any) -> AnyCallable:
 
 
 def make_literal_validator(type_: Any) -> Callable[[Any], Any]:
-    if sys.version_info >= (3, 7):
-        permitted_choices = type_.__args__
-    else:
-        permitted_choices = type_.__values__
+    permitted_choices: Tuple[Any, ...] = all_literal_values(type_)
```

Comment:
```suggestion
    permitted_choices = all_literal_values(type_)
```
I doubt this type hint is necessary.

Reply:
Done